server/src/com/cloud/ha/HighAvailabilityManagerImpl.java

```diff
@@ -497,7 +498,7 @@ protected Long restart(final HaWorkVO work) {
 
                 boolean fenced = false;
                 if (alive == null) {
-                    s_logger.debug("Fencing off VM that we don't know the state of");
+                    s_logger.debug("Fencing off VM " + vm + " that we don't know the state of");
```

Comment:
Turning all of those log debug lines without a `if` statement to lambda expressions when we require Java8 would be easier if there're not wrapped within a `if`:
```
s_logger.debug(() -> "Fencing off VM " + vm + " that we don't know the state of");
```
with
```
default void debug(Supplier<String> stringSupplier) {
    if (isDebugEnabled()) {
        debug(stringSupplier.get());
    }
}
```
The concatenation will be only done when debug is enabled.

You're right, I turn it to `warn`.